app/controllers/placements/support/schools/placements/add_placement_controller.rb

```diff
@@ -0,0 +1,61 @@
+class Placements::Support::Schools::Placements::AddPlacementController < Placements::ApplicationController
```

Comment:
```suggestion
class Placements::Support::Schools::Placements::AddPlacementController < Placements::Schools::Placements::AddPlacementController
```
Most of this code is identical to what's in the `Placements::Schools::Placements::AddPlacementController`

Reply:
+1 to Jamie's suggestion. I feel like the main (only?) things which differs are:

- `step_path` method
- `back_link_path` method
- and the redirect path in the `update` method, which could be extracted out to another overridable method